src/coins.cpp

```diff
@@ -9,6 +9,11 @@
 #include <random.h>
 #include <version.h>
 
+/* Check structure sizes are nicely packed */
+static_assert(sizeof(Coin) == 48, "unexpected sizeof(Coin)");
```

Comment:
I don't like statically asserting numbers here. It may differ per platform, architecture.